### PR TITLE
Adding winston logger

### PR DESCRIPTION
**Pull request by OUeasley**
Adds the winston remote logger and removes `console.log`

### CODE REVIEW

File: src/lib/logger.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    customFormat
+  ),
+  transports: [
+    new winston.transports.Console({
+      format: winston.format.simple(), // Using simple format for console readability
+    }),
+    new winston.transports.Http({
+
+      level: 'info', // Minimum level of logs to send
+      host: 'default.main.great-jarzyna-f1rlbvu.cribl-staging.cloud', // The remote server host
```
Comment (OUeasley) [inline src/lib/logger.ts:23]: Really the main change here, most everything else was Stream setup

File: package.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
     "react": "^18",
     "react-dom": "^18",
     "tailwind-merge": "^2.2.1",
-    "tailwindcss-animate": "^1.0.7"
+    "tailwindcss-animate": "^1.0.7",
```
Comment (OUeasley) [inline package.json:24]: No idea why this updated 🤷

**Review by OUeasley — COMMENTED** [review]
Just gonna merge but leaving the PR for posterity

File: src/lib/logger.ts (hunk 1 above)
Comment (jsakaicribl) [reply src/lib/logger.ts:23]: This is cool! I was able to see the log in the host.

**Pull request merged**